docs/en/04_Changelogs/4.8.0.md

```diff
@@ -10,47 +10,58 @@
+to drive various parts of the CMS UI, as well as building your own GraphQL APIs in your Silverstripe CMS projects.
 
-The CMS currently ships with `silverstripe/graphql:^3`, which is stable but slow for larger schemas.
-The module was rewritten from scratch with performance in mind
-(see [goals](https://github.com/silverstripe/silverstripe-framework/issues/8824) and [pull request](https://github.com/silverstripe/silverstripe-graphql/pulls)).
-The resulting`silverstripe/graphql:^4` release is currently in a alpha stage.
-We encourage everyone to get started, read our new [GraphQL documentation](https://docs.silverstripe.org/en/4/developer_guides/graphql/), 
-provide [feedback](https://github.com/silverstripe/silverstripe-graphql/issues) and help us stabilise the module.
+The CMS currently ships with `silverstripe/graphql:^3.5`, which is stable but slow for larger schemas.
+The module was rewritten from scratch with performance in mind.
 
-In order to facilitate the new module release, most GraphQL helper classes to drive the CMS UI
-have been marked as deprecated, and moved to `GraphQL/_legacy` folders within core modules.
-The GraphQL schema under `admin/graphql` has been altered slightly to enable forwards compability,
-with lower-camelcase field names, and a different filter argument structure.
-In the unlikely case that you are relying on those classes or queries directly,
-consider upgrading to `silverstripe/graphql:^4`.
-Your own GraphQL schemas outside of `admin/graphql` should not be affected by the changes
-if you choose to stay on `silverstripe/graphql:^3` for the time being.
+If you want to learn more about the development of `silverstripe/graphql` v4:
+- review [Epic: Scaling decoupled CMS usage (GraphQL performance)](https://github.com/silverstripe/silverstripe-framework/issues/8824) to better understand the goals
+- look at the [`silverstripe/graphql` pull requests](https://github.com/silverstripe/silverstripe-graphql/pulls) to see exactly what has changed.
+
+The resulting`silverstripe/graphql:^4` release is currently in an alpha stage.
 
 We are planning to require the use of `silverstripe/graphql:^4` in a future CMS 4.x release.
 
+#### Start using Silverstripe GraphQL v4 today
+
+We encourage everyone to get started working with GraphQL v4 today:
+- You can start by reading our new [GraphQL documentation](https://docs.silverstripe.org/en/4/developer_guides/graphql/).
+- You can provide [feedback](https://github.com/silverstripe/silverstripe-graphql/issues) and help us stabilise the module.
+
+If your project uses `silverstripe/recipe-cms`, it is still locked to `silverstripe/graphql:^3.5`.
+To use `silverstripe/graphql:^4`, you'll need to "inline" the `silverstripe/recipe-cms` requirements
+in your root `composer.json` and change `silverstripe/graphql` to `^4`.
+
+You can inline `silverstripe/recipe-cms` by running this command:
+```bash
+composer update-recipe silverstripe/recipe-cms
+```
```

Comment:
Recipe-plugin has a dedicated way of inlining recipes https://github.com/silverstripe/recipe-plugin#inlining-recipes

I think it makes more sense to encourage people to try this first.